Source/Flow/Private/Nodes/World/FlowNode_CallOwnerFunction.cpp

```diff
@@ -97,12 +97,31 @@ void UFlowNode_CallOwnerFunction::PostLoad()
 	FObjectPropertyBase* ParamsProperty = FindFProperty<FObjectPropertyBase>(GetClass(), GET_MEMBER_NAME_CHECKED(UFlowNode_CallOwnerFunction, Params));
 	check(ParamsProperty);
 
-	UClass* RequiredParamsClass = GetRequiredParamsClass();
-	if (IsValid(RequiredParamsClass))
+	// NOTE (gtaylor) This fixes corruption in FlowNodes that could have been caused with
+	//  a previous version of the code (which was inadvisedly calling SetPropertyClass)
+	//  to restore the correct PropertyClass for this node.  
+	//  (it could be removed in a future release, once all assets have been updated)
+	if (ParamsProperty->PropertyClass != UFlowOwnerFunctionParams::StaticClass())
+	{
+		ParamsProperty->SetPropertyClass(UFlowOwnerFunctionParams::StaticClass());
+	}
+}
+
+bool UFlowNode_CallOwnerFunction::CanEditChange(const FProperty* InProperty) const
+{
+	if (!Super::CanEditChange(InProperty))
```

Comment:
We're not allowing the user to choose the params object directly any longer (the way this was done previously was dangerous).  So now the params object is only created when selecting the function, and this CanEditChange prevents the user from choosing an incompatible params object.